src/request/remove-key/RemoveKeyApi.js

```diff
@@ -25,11 +25,14 @@ class RemoveKeyApi extends TopLevelApi { // eslint-disable-line no-unused-vars
         /** @type {HTMLElement} */
         const $removeKey = (document.getElementById(RemoveKeyApi.Pages.REMOVE_KEY));
         /** @type {HTMLElement} */
-        this.$privacy = (document.getElementById(RemoveKeyApi.Pages.PRIVACY_AGENT));
+        const $privacy = (document.getElementById(RemoveKeyApi.Pages.PRIVACY_AGENT));
         /** @type {HTMLElement} */
         const $showWords = (document.getElementById(RemoveKeyApi.Pages.SHOW_WORDS));
         /** @type {HTMLElement} */
-        this.$downloadKeyFile = (document.getElementById(RemoveKeyApi.Pages.DOWNLOAD_KEY_FILE));
+        const $downloadKeyFile = (document.getElementById(RemoveKeyApi.Pages.DOWNLOAD_KEY_FILE));
+
+        this.$privacy = $privacy;
+        this.$downloadKeyFile = $downloadKeyFile;
```

Comment:
Add comment here signaling that this is a workaround for the typescript typechecker.